From 850603effdefd56ad580e4ad51d55d22324b602e Mon Sep 17 00:00:00 2001
From: Jonas Larsson <ljonas@google.com>
Date: Tue, 26 Feb 2019 16:55:43 -0800
Subject: [PATCH 1/1] Reduce default number of extra decoder output buffers
 from 3 to 0.

1080p playback is still fine and we save tons of CMA memory.
Any user that needs more output buffers can just set the
frame-plus property.

Change-Id: I76b953bf8744d290129557c10ee7e6ac27130811
---
 plugins/vpu/gstvpudec.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/vpu/gstvpudec.c b/plugins/vpu/gstvpudec.c
index 600a5f9..dc698c4 100755
--- a/plugins/vpu/gstvpudec.c
+++ b/plugins/vpu/gstvpudec.c
@@ -61,7 +61,7 @@ enum
 #define DEFAULT_LOW_LATENCY FALSE
 #define DEFAULT_OUTPUT_FORMAT 0
 #define DEFAULT_ADAPTIVE_FRAME_DROP TRUE
-#define DEFAULT_FRAMES_PLUS 3
+#define DEFAULT_FRAMES_PLUS 0
 /* Default to use VPU memory for video frame buffer as all video frame buffer
  * must registe to VPU. Change video frame buffer will cause close VPU which
  * will cause video stream lost.
-- 
2.26.0

